api/models/user_group.py

```diff
@@ -0,0 +1,32 @@
+from api.mixins import TimestampMixin
+from api import db, helpers
+from flask_sqlalchemy import BaseQuery
+
+
+class UserGroupQuery(BaseQuery):
+    def get_by_uuid(self, uuid):
+        return self.filter(UserGroup.group_uuid == uuid).first()
+
+    def get_unique(self, uuid, phone):
```

Comment:
We are capturing group uuid from RP. So to get unique group details, we need two element. We are using user phone and group to get these details.